welcome.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  <head> 
+    <link rel="stylesheet" href="/welcome.css" />
+<link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:ital,wght@0,700;1,700&display=swap" rel="stylesheet">
+   <style>
+     body {
+  background-image: url('https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2FIMG_9976.PNG?v=1604357962085');
+  background-size: 200%;
+     }
+    </style>
+    <title>welcome page!</title>
+    
+  </head>  
+  
+  <body>
+        <div class="start">
+
+          <h1>START BUILDING</h1></div>
```

Comment:
Use `text-transform: uppercase` in CSS here instead of typing this out in capital letters